_IBC/bandchain-comdex.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "bandchain",
+    "client_id": "07-tendermint-124",
+    "connection_id": "connection-107"
+  },
+  "chain_2": {
+    "chain_name": "comdex",
+    "client_id": "07-tendermint-81",
+    "connection_id": "connection-48"
+  },
+  "channels": [
+    {
+      "chain_1": {
+        "channel_id": "channel-96",
+        "port_id": "oracle"
+      },
+      "chain_2": {
+        "channel_id": "channel-43",
+        "port_id": "bandoracleV1"
+      },
+      "ordering": "unordered",
+      "version": "bandchain-1",
```

Comment:
not sure that bandchain-1 is a valid version value; it's meant to be one of the ibc standards, like ics20. Perhaps ics31 (for Cross-chain Queries) might apply? I could be wrong...

Reply:
bandchain-1 is the channel version for fetching oracle price specified by band